cmake/bintools/gnu/target_memusage.cmake

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Add and/or prepare print of memory usage report
+#
+# Usage:
+#   toolchain_bt_output_print_mem_usage(
+#     RESULT_CMD_LIST    <List of commands to be executed, usually after build>
+#     RESULT_BYPROD_LIST <List of command output byproducts>
+#   )
+#
+macro(toolchain_bt_output_print_mem_usage)
+  cmake_parse_arguments(
+    # Prefix of output variables
+    TOOLCHAIN_BT_O_P_MEM_USAGE
+    # List of argument names without values, hence boolean
+    ""
+    # List of argument names with one value
+    "RESULT_CMD_LIST;RESULT_BYPROD_LIST"
+    # List of argument names with multiple values
+    ""
+    # Parser input
+    ${ARGN}
+  )
+
+  # Here we make use of the linkers ability to produce memory usage output
+  #  and thus we have no need for the above provided arguments, but another
+  #  toolchain with a different set of binary tools, most likely will...
```

Comment:
If we're not using the arguments we shouldn't be calling cmake_parse_arguments to parse them.

Reply:
It was meant to show how to make use of the parameters, for a toolchain where the bintools cannot make use of the linker flag.
Do we then simply leave it commented out?
Or should it be removed completely?

Reply:
Remove it completely.

Reply:
Ok, will do.